### DISCUSSION TITLE
Installed as a PWA: How to open links in new browser?

### DISCUSSION THREAD

**Discussion by ChappIO** [coder/code-server#5990] [Q&A]
So here’s my setup:

1. I’ve installed code-server on a VM
2. I have local access to the VM (through VPN)
3. I have installed code-server as a PWA on my iPad
4. I have started a NodeJS application from the terminal in code-server

It detects the new port correctly and allows me to open it. However, it opens the app inside the PWA, completely closing my code-server workspace.

Is there a known way to have code-server open the app in a safari (or default) browser?

I attached a screen recording: As you can see the app is opened on top over code-server instead of in a new browser window.

https://user-images.githubusercontent.com/13708940/215261668-1259df05-503e-4e5a-beff-99577ad827f6.MOV

**Comment by code-asher** [comment]
Interesting!  We will need to try reproducing in Codespaces, I 
think it is likely this is an issue with upstream Code web.
